wp-modules/editor/tests/ModelTest.php

```diff
@@ -128,7 +128,7 @@ public function test_slug_and_filename_stay_the_same_after_content_update() {
 		$post->post_title   = 'This title remains the same after content changes.';
 		$post->post_content = 'test pattern content';
 		$post->post_status  = 'publish';
-		$post->post_name    = 'this-slug-remains-the-same-after-content-changes';
+		$post->post_name    = 'remains-the-same-after-content-changes';
```

Comment:
Hope it's OK I changed this, as it's not only the slug.

Reply:
Can you elaborate on that? What is it other than the slug?

Reply:
It's also the filename (the filename is saved as the `->post_name`, where the pattern slug is never in the WP_Post).

Happy to revert this, it's a nitpick of mine.